core/src/main/java/org/kiwix/kiwixmobile/core/utils/dialog/KiwixDialog.kt

```diff
@@ -22,10 +22,11 @@ import android.app.Activity
 import android.view.View
 import org.kiwix.kiwixmobile.core.R
 
+@Suppress("LongParameterList")
 sealed class KiwixDialog(
   val title: Int?,
   val message: Int?,
-  val positiveMessage: Int,
+  val positiveMessage: Int?,
```

Comment:
Having a not-optional field is a good idea to make sure the user knows what's going on. It feels weird if we don't have any message to say users what's going on. What do you think?

Reply:
Good point :+1: